apps/wolfsshd/configuration.c

```diff
@@ -503,49 +505,80 @@ static int HandleInclude(WOLFSSHD_CONFIG *conf, const char *value)
             prefixLen = (int)(ptr - value);
         }
 
-        d = opendir(path);
-        if (d) {
-            while ((dir = readdir(d)) != NULL) {
-                if (dir->d_type == DT_DIR) {
-                    /* Skip sub-directories */
-                    continue;
+        if (!WOPENDIR(NULL, conf->heap, &d, path)) {
```

Comment:
Let's change the `memcpy`s above to `WMEMCPY`s. Also move the `} else {` onto a new line.

Reply:
Check